app/services/symphony_client.rb

```diff
@@ -195,11 +233,19 @@ def patron_linked_resources_fields(item_details = {})
       end
     end
 
+    def hold_info_request
+      authenticated_request "/circulation/holdRecord/key/#{hold_key}",
+                            headers: { 'x-sirs-sessionToken': session_token },
+                            params: {
+                              includeFields: '*,item{*,bib{shadowed,title,author},call{*}}'
+                            }
+    end
+
```

Comment:
```suggestion
```

I could not find where this method is used? maybe I am missing it?

Reply:
no it isn't sorry and thanks